deployments/charts/kuma/README.md

```diff
@@ -58,6 +58,10 @@ The chart supports Helm v3+.
 All Kuma CRDs are loaded via the [`crds`](crds) directory. For more detailed information on CRDs and Helm,
 please refer to [the Helm documentation][helm-crd].
 
+## Note to Chart developers
+
+The charts are used internally in `kumactl install`, therefore the following rules apply when developing new chat features:
+ * all templates that start with `pre-` and `post-` are omitted when processing in `kumactl install`  
```

Comment:
Maybe add a note that you need to run `make generate...` to have consistent HELM and kumactl install?

Reply:
Makes sense, but I think there is still this verification part of `make test`